test/kinematic/vector/acceleration_of_rotating_body_test.py

```diff
@@ -0,0 +1,136 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    convert_to,
+    Quantity,
+    SI,
+    QuantityVector,
+)
+from symplyphysics.laws.kinematic.vector import acceleration_of_rotating_body as acceleration_law
+
+# Description
+## A body is moving along a curve. At a certain point in time, its radial acceleration
+## with respect to the momentary rotational axis is (-1, 0, 0.4) m/s^2, and its tangential
+## acceleration is (0, 0.1, -3) m/s^2. Its acceleration should amount to (-1.0, 0.1, -2.6) m/s^2.
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    a_r = QuantityVector([
+        Quantity(-1.0 * units.meter / units.second**2),
+        Quantity(0.0 * units.meter / units.second**2),
+        Quantity(0.4 * units.meter / units.second**2),
+    ])
+    a_t = QuantityVector([
+        Quantity(0.0 * units.meter / units.second**2),
+        Quantity(0.1 * units.meter / units.second**2),
+        Quantity(-3.0 * units.meter / units.second**2),
+    ])
+    a_total = QuantityVector([
+        Quantity(-1.0 * units.meter / units.second**2),
+        Quantity(0.1 * units.meter / units.second**2),
+        Quantity(-2.6 * units.meter / units.second**2),
+    ])
+    Args = namedtuple("Args", "a_r a_t a_total")
+    return Args(a_r=a_r, a_t=a_t, a_total=a_total)
+
+
+def test_basic_law(test_args):
+    result = acceleration_law.calculate_acceleration(test_args.a_r, test_args.a_t)
+    assert len(result.components) == 3
+    assert SI.get_dimension_system().equivalent_dims(result.dimension, units.acceleration)
+    correct_values = (-1.0, 0.1, -2.6)
```

Comment:
You can take correct values from the `test_args`

Reply:
It's just that you told me earlier to avoid extra conversions via `convert_to` so I went with this

Reply:
Yes but you should have a single point of truth. Before you didn't have total vector in test_args so this way you could avoid extra conversions. Now you have and you've got a copypaste - 2 sources of data. It is error prone and should be avoided.

Reply:
I agree with you on this.